test/unit/core/reducers/index.ts

```diff
@@ -100,18 +100,36 @@ suite('reducers', ({ expect, stub }) => {
     expect(reducer(oldState, { type: Actions.REFRESH_STATE, payload }).data.past).to.eql([]);
   });
 
-  it('should advance history on SAVE_STATE', () => {
-    const updated = reducer(<any>{
-      data: {
-        past: [],
-        present: {},
-        future: []
-      }
-    }, <any>{ type: Actions.SAVE_STATE });
+  describe('undo', () => {
+    const state = { a: 'b' };
+    const present = { c: 'd' };
+
+    beforeEach(() => {
+      stub(dataReducers, 'default').returns(state);
+    });
+
+    reducers.HISTORY_UPDATE_ACTIONS.forEach((action) => {
+      it('should add to the past object on HISTORY_UPDATE_ACTIONS', () => {
```

Comment:
Change this string to use backticks and replace `HISTORY_UPDATE_ACTIONS` with `${action}`. We have no way of knowing which test this is otherwise.